.github/pull_request_template.md

```diff
@@ -0,0 +1,51 @@
+<!--- indicate the protocol code in the Title above -->
+
+## Task list
+
+<!--see https://docs.github.com/en/github/managing-your-work-on-github/about-task-lists
+for an explanation on how to use task lists-->
+
+Steps by contributor:
+
+- [ ] Add description to this pull request (see #-description)
+- [ ] Submit the pull request by clicking 'create pull request'
+- [ ] Mark the pull request as draft
+- [ ] Check if the version number in the `index.Rmd` yaml section is of format `yyyy.nn.dev`
+- [ ] In case of a protocol created from a pre-existing `docx` protocol, check if all sections comply with current template for a new protocol
+- [ ] Add further commits and push commits to GitHub if needed
+- [ ] Update the protocol-specific `NEWS.Rmd`
+- [ ] Mark the pull request as 'ready for review'
+
+
+Review steps:
+
+- [ ] Add reviewers, at least one subject-matter specialist and one administrator
+- [ ] Wait for review comments and address them
+- [ ] Iterate until reviewer approvals (merging the pull request will be done by an administrator)
+
+To be done by an administrator after review, but before merging the pull request:
+
+- [ ] Update the `.zenodo.json` file (authorship will extend as protocols are added)
+- [ ] Bump the version number in the `index.Rmd` yaml section from `yyyy.nn.dev` to `YYYY.NN`.
+- [ ] Update the repo `NEWS.md` file
+
+To be done by an administrator after merging this pull request:
+
+- [ ] Add general tag to the merge commit (`protocols-YYYY.NN`)  (see [release model](README.md#release-model))
+- [ ] Add specific tag to the merge commit (`protocol-code-YYYY.NN`)  (see [release model](README.md#release-model))
```

Comment:
```suggestion
- [ ] Add specific tag to the merge commit (`<protocol-code>-YYYY.NN`)  (see [release model](README.md#release-model))
```

I suggest to do this in README as well. `protocol-code` is a variable, whereas in `protocols-YYYY.NN` it is a string, which is confusing to the new reader. I know, this also holds for YYYY and NN, but I think this is more clear.